src/plugins/intel_cpu/src/utils/debug_capabilities.cpp

```diff
@@ -506,11 +506,6 @@ std::ostream & operator<<(std::ostream & os, const PrintableModel& model) {
         os << std::endl;
 
         // recursively output subgraphs
```

Comment:
Could you please elaborate a little bit on the reason of removing the case `if (auto subgraph = std::dynamic_pointer_cast<ngraph::snippets::op::Subgraph>(op))`? Is `if (auto msubgraph = std::dynamic_pointer_cast<op::util::MultiSubGraphOp>(op))` enough for both the cases?

Reply:
Yes, now `ngraph::snippets::op::Subgraph` is derived from `ov::op::util::SubGraphOp` which is derived from `MultiSubGraphOp`, so print `MultiSubGraphOp` is enough.